src/spikeinterface/sorters/internal/si_based.py

```diff
@@ -21,7 +21,11 @@ def _setup_recording(cls, recording, output_folder, params, verbose):
     @classmethod
     def _get_result_from_folder(cls, output_folder):
         sorting = load(output_folder / "sorting")
-        return sorting
+        from spikeinterface.core.numpyextractors import SharedMemorySorting
+
+        shm_sorting = SharedMemorySorting.from_sorting(sorting)
+        del sorting
+        return shm_sorting
```

Comment:
I do not think we should use the sharedmem always.
Whey do we need this ?

Reply:
I was trying to get rid of the memmap, but @zm711  found the patch, and thought it could not hurt anything to leave that here, can it?

Reply:
This has been removed